### PR TITLE
Mvh highest total score

### PR DESCRIPTION
**Pull request by mikez321**
Creates some fixture files, and also half of the methods in it2

### CODE REVIEW

File: fixture_files/game_teams_fixture.csv [hunk 1]
```diff
@@ -0,0 +1,65 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (aperezsantos) [inline fixture_files/game_teams_fixture.csv:1]: Nice work on the fixture files. Will definitely help our method building.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -21,4 +21,31 @@ def load_team_data
...
+  end
+
+  def percentage_ties
+    ties = game_collection.games.find_all do |game|
+      game.home_goals == game.away_goals
+    end.length
+    (ties / game_collection.games.length.to_f).round(2)
+  end
+
+  def count_of_games_by_season
```
Comment (aperezsantos) [inline lib/stat_tracker.rb:53]: Would a reduce method improve this?

Comment (mikez321) [reply lib/stat_tracker.rb:53]: I went with hash.new to set a default value.  I don't think Reduce would be too much more helpful and this was easier.  Maybe a refactor later though!

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -11,35 +11,56 @@
...
+    assert_instance_of TeamCollection, @stattracker.team_collection
+    assert_equal Team, @stattracker.team_collection.teams.first.class
+  end
+
+  def test_it_can_return_the_highest_total_score
+    assert_equal 5, @stattracker.highest_total_score
+  end
+
+  def test_it_can_return_the_biggest_blowout
+    assert_equal 5, @stattracker.biggest_blowout
```
Comment (aperezsantos) [inline test/stat_tracker_test.rb:49]: This is just a reminder for us to rename the stattracker instance

**Review by aperezsantos — APPROVED** [review]

**Review by colintalex — APPROVED** [review]

**Pull request merged**